# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qmatplot/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./src/ksmatrix.h.orig	2005-03-03 11:22:41.655965224 -0300
+++ ./src/ksmatrix.h	2005-03-03 11:23:09.674705728 -0300
@@ -332,8 +332,8 @@
 class KSMatrixFormula : public QSMatrix
  {
 	Q_OBJECT
-	Q_PROPERTY( QString formula READ formula WRITE setFormula );
-	Q_PROPERTY( bool transposition READ transposition WRITE setTransposition );
+	Q_PROPERTY( QString formula READ formula WRITE setFormula )
+	Q_PROPERTY( bool transposition READ transposition WRITE setTransposition )
 
   public:
 	/**
--- ./src/qscobjects.h.orig	2005-03-03 12:05:35.000000000 -0300
+++ ./src/qscobjects.h	2005-03-03 12:07:30.513196792 -0300
@@ -246,7 +246,7 @@
 	QSGArrow m_end2style;
 	QSGPoint m_end1point;
 	QSGPoint m_end2point;
-	QSGLine m_line;;
+	QSGLine m_line;
    };
 
 //-----------------------------------------------------------------------//
--- ./src/kspanelmanager.cpp.orig	2005-03-03 13:00:05.446574016 -0300
+++ ./src/kspanelmanager.cpp	2005-03-03 13:00:25.259561984 -0300
@@ -47,7 +47,7 @@
 	 Qt::Orientation	o;
 
          qmatplotShell  	*shell;
-	 KSWorkbook 		*workbook;;
+	 KSWorkbook 		*workbook;
 	 KSPanel 		*curr_panel;
 
 	 QSplitter		*splitter;
--- ./src/kswinworksheet.h.orig	2005-03-03 13:03:02.000000000 -0300
+++ ./src/kswinworksheet.h	2005-03-03 13:03:28.000000000 -0300
@@ -40,7 +40,7 @@
   public:
 	KSWinWorksheet( KSWorkspace *workspace, bool worksheetMode );
 	virtual ~KSWinWorksheet();
-	void KSWinWorksheet::setDataObject( QSData *data );
+	void setDataObject( QSData *data );
 	KSChannelList *chanList() const;
 	KSChannelListItem *selectedItem() const;
 	KSMPanel *settingsPanel() const;
